build/cite/Makefile

```diff
@@ -18,18 +18,14 @@ clean:		## Will Clean the Environment of previous runs.
 	docker-compose down --remove-orphans --rmi all -v
 	docker-compose rm -vfs
 	@echo Cleanning the previous logs.
-	sudo rm -Rf logs/
+	rm -Rf logs/*
 
 build: $(suite)		## Will Build the GeoServer Docker Image for the Environment.
 	@echo Build the GeoServer Docker Image
 	docker-compose -f docker-compose.yml -f ./$(suite)/docker-compose.override.yml \
       build --build-arg GEOSERVER_WEBAPP_SRC=${war_url} geoserver
 
 test: $(suite)		## Will running the Suite test with teamengine.
-    @echo Getting the GeoServer version.
```

Comment:
why this change?

Reply:
I had some errors when run in the CI, so I take it out, later I will find a way of added it without any problem.

Reply:
I'll wait for the fix to merge the PR

Reply:
it's fixed in https://github.com/randomorder/geoserver/pull/7/commits/a360566ca11d2839cca6e3f9800abe9c6915a40a